drivers/dma/dma_stm32u5.c

```diff
@@ -24,9 +24,13 @@ LOG_MODULE_REGISTER(dma_stm32, CONFIG_DMA_LOG_LEVEL);
 
 #define DT_DRV_COMPAT st_stm32u5_dma
 
-/* STM32U5 soc has only one GPDMA instance of 15 channels */
+/* STM32U5 soc has only one GPDMA instance of 16 channels */
```

Comment:
Question on the comment: Is it the GPDMA IP that can have up to 16 channels, or the U5 IP ?